### PR TITLE
Implement #7569: SelectManyMenu: provide PF Selenium - wrapper and integrationtests

### LINKED ISSUE #7569: SelectManyMenu: provide PF Selenium - wrapper and integrationtests
I´ll try to work on this.

### PR DESCRIPTION
**Pull request by christophs78**

### CODE REVIEW

File: primefaces-selenium/primefaces-selenium-components/src/main/java/org/primefaces/selenium/component/SelectManyMenu.java [hunk 1]
```diff
@@ -49,19 +49,27 @@
...
+    public void select(WebDriver webDriver, String label, boolean withMetaKey) {
+        if (isSelected(label)) {
             return;
         }
 
-        clickOnListItemWithMetaKey(webDriver, label, keepExistingSelection);
+        toogleSelection(webDriver, label, withMetaKey);
+    }
+
+    public void toogleSelection(WebDriver webDriver, String label, boolean withMetaKey) {
```
Comment (melloware) [inline primefaces-selenium/primefaces-selenium-components/src/main/java/org/primefaces/selenium/component/SelectManyMenu.java]: `toggle` not `toogle`

**Review by melloware — CHANGES_REQUESTED** [review]

File: primefaces-selenium/primefaces-selenium-components/src/main/java/org/primefaces/selenium/component/SelectManyMenu.java (hunk 1 above)
Comment (christophs78) [reply primefaces-selenium/primefaces-selenium-components/src/main/java/org/primefaces/selenium/component/SelectManyMenu.java]: thanks for pointing be on this typo!

**Comment by tandraschko** [comment]
why do you pass WebDriver webDriver everywhere? just use getWebDriver from inheritance

**Comment by christophs78** [comment]
> why do you pass WebDriver webDriver everywhere? just use getWebDriver from inheritance

Thanks for pointing me on this!

**Review by melloware — APPROVED** [review]

**Pull request merged**